src/main/java/de/retest/web/ScreenshotProvider.java

```diff
@@ -0,0 +1,26 @@
+package de.retest.web;
+
+import java.awt.image.BufferedImage;
+
+import org.openqa.selenium.WebDriver;
+
+import com.assertthat.selenium_shutterbug.core.Shutterbug;
+import com.assertthat.selenium_shutterbug.utils.web.ScrollStrategy;
+
+public class ScreenshotProvider {
+
+	private static final int SCROLL_TIMEOUT = 100;
+	private static final boolean USE_DEVICE_PIXEL_RATIO = true;
+
+	public static BufferedImage shootFullPage( final WebDriver driver ) {
+		if ( driver.getClass().getSimpleName().equals( "ChromeDriver" ) ) {
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Use an "instanceof" comparison instead. ([squid:S1872](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS1872))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=101&issues=AWcNHOHWptIL6UVyV_nE&open=AWcNHOHWptIL6UVyV_nE)

[ISSUE_KEY]: AWcNHOHWptIL6UVyV_nE